guides/doc-Configuring_Load_Balancer/topics/Configuring_Capsule_Server_with_Default_SSL_Certificates_for_Load_Balancing_with_Puppet.adoc

```diff
@@ -82,6 +90,13 @@ This command creates the following files on the system where you configure {Smar
 * `/etc/puppetlabs/puppet/ssl/certs/_{smartproxy-example-com}_.pem`
 * `/etc/puppetlabs/puppet/ssl/private_keys/_{smartproxy-example-com}_.pem`
 * `/etc/puppetlabs/puppet/ssl/public_keys/_{smartproxy-example-com}_.pem`
+.. Resume the Puppet agent:
```

Comment:
You're starting the server, not the agent.

Reply:
Fixed